### PR TITLE
Comment added to sleepy pen define

### LINKED ISSUE #7118: Someone please update the parapen info
The in-game description of the parapen and the Wiki's description differ greatly and I'm pretty sure neither is correct. I don't know its function well enough to update them, but someone who actually uses it surely could.

### PR DESCRIPTION
**Pull request by Intigracy**
Sleepy pens might or might not be refillable, I haven't looked at syringe or beaker attack code, but there's no attackby code for it so.

Parapens never were.

Fixes #7118

### CODE REVIEW

**Comment by Exxion** [comment]
This does not even slightly come close to so much as touching the issue #7118 is referring to

**Comment by ghost** [comment]
False advertising

**Comment by Intigracy** [comment]
"The in-game description of the parapen and the Wiki's description differ greatly and I'm pretty sure neither is correct. I don't know its function well enough to update them, but someone who actually uses it surely could."

The ingame description for the parapen in the uplink does NOT say it is refillable. I updated the wiki.

What else didn't I do that you were expecting?

**Comment by Exxion** [comment]
Refillable isn't at all the only issue. Almost everything about both the in-game description and the Wiki description is wrong, including
1. What it contains
2. How many uses it has
3. If there were any more information it would also be wrong

**Comment by Intigracy** [comment]
You should read the wiki again before you continue commenting.

**Comment by Intigracy** [comment]
And the ingame description.

"A syringe disguised as a functional pen, filled with a neuromuscular-blocking drug that renders a target immobile on injection and makes them seem dead to observers. Side effects of the drug include noticeable drooling. The pen holds one dose of paralyzing agent, and cannot be refilled."

Chloral hydrate doesn't cause drooling, which is what's in the sleepypen. Chloral hydrate makes someone asleep, it doesn't make them seem dead. The contents of the parapen, however, does all of this.

By the way, this exact same description was on the wiki before I even touched it.

fuckyouceltfuckyouex



tl;dr fuck you celt go fix ventcrawling

**Comment by Exxion** [comment]
I'm not sure why you're fixating on sleepypens as the issue is (or was, with the Wiki updated) with parapens.

I'm not really sure why I thought the in-game description was wrong

And by the way, you might want to mention in the OP that you fixed the Wiki, considering that's what _actually_ fixed the issue and the PR itself has nothing to do with it

**Comment by Intigracy** [comment]
I mentioned to you in the comment I screencapped that I updated the wiki, and you continued going on about it.

**Comment by Exxion** [comment]
I saw that
I misspoke

**Comment by Intigracy** [comment]
Now do me a favor and close your issue since it was addressed, as I've said numerous times.
